tests/system/test-api-search-post.js

```diff
@@ -1,53 +1,44 @@
 const test = require('ava')
-const { promisify } = require('util')
-const fs = require('fs')
-const path = require('path')
+
 const { deleteAllIndices, refreshIndices } = require('../helpers/es')
 const { randomId } = require('../helpers/utils')
 const ingest = require('../../src/lib/ingest')
 const intersectsGeometry = require('../fixtures/stac/intersectsGeometry.json')
 const stream = require('../../src/lib/esStream')
 const systemTests = require('../helpers/system-tests')
 
-const readFile = promisify(fs.readFile)
-
-/**
- * @param {string} filename
- * @returns {Promise<unknown>}
- */
-const loadJson = async (filename) => {
-  const filePath = path.join(__dirname, '..', 'fixtures', 'stac', filename)
-
-  const data = await readFile(filePath, 'utf8')
-  return JSON.parse(data)
+const ingestEntities = async (fixtures) => {
+  await ingest.ingestItems(
+    await Promise.all(fixtures.map((x) => systemTests.loadJson(x))), stream
+  )
+  await refreshIndices()
 }
 
 test.before(async (t) => {
   await deleteAllIndices()
-  const standUpResult = await systemTests.setup()
 
-  t.context = standUpResult
+  t.context = await systemTests.setup()
 
-  const fixtureFiles = [
-    'catalog.json',
+  // ingest collections before items so mappings are applied
```

Comment:
the underlying reason to separate the collection ingest and the item ingest is that when they're together, the bulk ingest would insert an item before the collection, which means that the ES index isn't created with the custom type mapping.

Reply:
This has something to do changing the sort to be by `properties.datetime,id,collection` instead of just `properties.datetime`, and that it somehow causes the query by ID to hang. 🤷🏻